### DISCUSSION TITLE
Bubble card - timer/countdown

### DISCUSSION THREAD

**Discussion by Mewerik123** [Clooos/Bubble-Card#660] [Questions about config, custom styles and templates]
Hi, I really like the Bubble card concept. I would like to ask if it is possible to set the time after the button is turned on, how long it should be on. I use it to turn on the garden watering and I often forget to turn it off. I'm a beginner with HA, can you help me please? Thank you very much.

**Comment by MrBearPresident** [comment]
See: #937
